sources/service-dns-cloudflare/main.tf

```diff
@@ -22,10 +22,15 @@ resource "cloudflare_record" "active_domain" {
 }
 
 resource "cloudflare_record" "passive_domain" {
-  for_each = local.passive_domains
+  for_each = {for y in local.passive_domains : y.name => {
+      host_tmp = split(".", y.name)
+      value = y.value
+      proxied = y.proxied
+      hosted_zone_id = y.hosted_zone_id
+  }}
 
   zone_id = each.value.hosted_zone_id
-  name    = "passive.${each.value.name}"
+  name    = join(".", concat(["${substr(each.value.host_tmp[0], 0, 56)}-passive"], length(each.value.host_tmp) > 1 ? slice(each.value.host_tmp, 1, length(each.value.host_tmp)) : []))
```

Comment:
Can we add a comment above this lines that clarifies why `56` is used as the substring length?